data/js/configuring-data.md

```diff
@@ -0,0 +1,164 @@
+# Configuring Data
+
+###### The `api.json` and `api-*.json` files allow us to configure the accessible paths for each data service.
+
+<!-- <article id="undestanding-configuration-files"> -->
+
+## Understanding configuration files
+
+By default WeDeploy Data service is going to use all the JSON files starting with `api-*` and also the file `api.json`.
+
+These files are used to help you manage features such as path validation, authentication, and params validation.
+
+The api JSON files are located at the same path of the `./container.json` and are used following the ordering filesystem.
+
+By the default, if you don't add any api.json file all the collections are going to be exposed.
+
+<!-- </article> -->
+
+<!-- <article id="json-attributes"> -->
+
+## JSON attributes
+
+After understanding how the api configuration files work, it's time to learn what are the supported attributes:
+
+<table class="table">
+  <tr>
+    <th>Field</th> <th>Description</th>
+  </tr>
+  <tr>
+    <td>path</td> <td>The path that represents the collection used to handle the request data.</td>
```

Comment:
The path if the full resource path that we want to access/modify with this request. We may describe here the pattern: `/collectionName/id/field/nestedField/...`